tests/integrations/tests/test_anthropic.py

```
@@ -671,17 +671,21 @@ def test_14_list_models(self, anthropic_client, test_config):
                 assert len(prev_response.data) <= 2
 
     @skip_if_no_api_key("anthropic")
-    def test_15_extended_thinking(self, anthropic_client, test_config):
+    @pytest.mark.parametrize("provider,model", get_cross_provider_params_for_scenario("thinking"))
+    def test_15_extended_thinking(self, anthropic_client, test_config, provider, model):
```

Ruff (0.14.8)

Unused method argument: `test_config`

(ARG002)